### PR TITLE
Update casing of IMUTracker to version 2

### PR DESCRIPTION
**Pull request by y6tada**
I designed new casing for IMUTracker with Autodesk Inventor. Also full assemble cad model of QUIKS and other parts for assembling are included in those commits.
In this update, connector direction issue that previous casing "Housing.SLDPRT" have had fixed, and the previous one renamed with "IMUTracker_casing_v1.SLDPRT".
Also, I attached appearance image to Hardware/ and displayed it on README, hope you like it though...🤔

### CODE REVIEW

**Comment by komori-t** [comment]
I have some request

1. Change directory structure
```
Hardware - IMUTracker
         └ IMUTracker_casing
```
into
```
Hardware - IMUTracker - Circuit
                      └ Casing
```

2. Upload IMUTracker_assemble.png to gist
3. Create new README.md at Hardware/IMUTracker
4. Refer IMUTracker_assemble.png from Hardware/IMUTracker/README.md
5. Update .gitattribute file

**Comment by y6tada** [comment]
According to your requests,

1. I changed directory str, also I added "IMUTracker_assemble.stp" into Hardware/IMUTracker/Casing/STEP

1.

**Comment by y6tada** [comment]
According to your requests,

1. I changed directory str, also I added "IMUTracker_assemble.stp" into Hardware/IMUTracker/Casing/STEP
2. Remove appearance image form TOP README, and remove .png from Hardware/
Then I uploaded image into my gist [here](https://gist.github.com/y6tada/de85adfb4af9ef8d579c89646622f2fe#file-image
)
3. Created new README at Hardware/IMUTracker
Do I need to attach some specific license for Hardware resources? If my understanding is correct, my resources are currently under the license of this entire project (Apatch). If so, that is totally no problem for me.
4. I did displayed images on Hardware/README
5. Updated .gitattribute
Then, I actually modified previous attribute you wrote, please confirm how it works.

Regards,

**Review by komori-t — CHANGES_REQUESTED** [review]
Some files (like .SLDPRT) are not specified as binary.
Specifying directories is easier.
Also, put LF on EOF.

**Comment by y6tada** [comment]
Now I specified "Hardware/IMMITracker/Casing/**" as binary.
Also added LF at the end line of .gitattributes

File: .gitattributes [hunk 1]
```diff
@@ -1,4 +1,2 @@
 "Unity/Demo/Assets/Robot Kyle/**/*" binary
-Hardware/IMUTracker_casing/**/* binary
-Hardware/IMUTracker/IMUTracker.step binary
-
+"Hardware/IMUTracker/Casing/**" binary
```
Comment (komori-t) [inline .gitattributes]: correct wildcard usage is:
`Hardware/IMUTracker/Casing/**/*`

**Review by komori-t — CHANGES_REQUESTED** [review]

**Review by komori-t — APPROVED** [review]
LGTM

**Pull request merged**